components/Menu/Menu.jsx

```diff
@@ -0,0 +1,111 @@
+// Generated with scripts/create-component.js
+import PropTypes from 'prop-types';
+import MaterialMenu from '@mui/material/Menu';
+import MenuItem from '@mui/material/MenuItem';
+import { createTheme } from '@mui/material/styles';
+import { baseFontSize, colors } from '../shared/theme';
+
+const materialThemeOverride = createTheme({
+  typography: {
+    fontFamily: `"Nunito Sans", sans-serif`,
+    fontSize: baseFontSize,
+    allVariants: {
+      color: colors.neutral[700],
+    },
+  },
+  palette: {
+    text: {
+      primary: colors.neutral[700],
+    },
+  },
+});
+
+const Menu = (props) => {
+  const {
+    open,
+    anchorEl,
+    anchorOrigin,
+    transformOrigin,
+    onClose,
+    items,
+    keepMounted,
+  } = props;
+
+  const itemOnClick = (item) => {
+    const { onClick } = item;
+    onClick();
+    onClose();
+  };
+  return (
+    <MaterialMenu
+      theme={materialThemeOverride}
+      aria-labelledby="button"
+      anchorReference="anchorEl"
+      anchorEl={anchorEl}
+      open={open}
+      onClose={onClose}
+      onBlur={onClose}
+      anchorOrigin={anchorOrigin}
+      transformOrigin={transformOrigin}
+      keepMounted={keepMounted}
+    >
+      {items.map((item) => (
+        <MenuItem
+          key={item.id}
+          onClick={() => itemOnClick(item)}
+          theme={materialThemeOverride}
+          color="textPrimary"
+        >
+          {item.content}
+        </MenuItem>
+      ))}
+    </MaterialMenu>
+  );
+};
+
+Menu.propTypes = {
+  /** If true, the component is shown. */
+  open: PropTypes.bool,
+  /** Menu contents, has a content parameter, an id and a handleClick function. */
+  items: PropTypes.arrayOf(
+    PropTypes.shape({
+      id: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
+      content: PropTypes.ReactNode,
```

Comment:
```suggestion
      id: PropTypes.oneOfType([PropTypes.string, PropTypes.number]).isRequired,
      content: PropTypes.node.isRequired,
```

Reply:
PropTypes.ReactNode causes typing error in console.
![image](https://github.com/catho/quantum/assets/105747728/22e01be8-5a8a-43d9-af63-3452a80d86e7)